renderer/components/Onboarding/Onboarding.js

```diff
@@ -177,8 +183,12 @@ class Onboarding extends React.Component {
           <Wizard.Step key="Autopilot" component={Autopilot} {...{ autopilot, setAutopilot }} />,
           <Wizard.Step
             key="BackupSetup"
+            canSkip
             component={BackupSetup}
+            hideSkipBackupDialog={hideSkipBackupDialog}
             isRestoreMode
+            isSkipBackupDialogOpen={isSkipBackupDialogOpen}
+            onSkip={showSkipBackupDialog}
```

Comment:
This is being passed but not used since `BackupSetup` defines its own `onSkip` handler.

Reply:
My bad - realised this is to influence the wizard button behaviour.